microceph/cmd/microceph/disk_add.go

```diff
@@ -14,29 +21,32 @@ type cmdDiskAdd struct {
 	common *CmdControl
 	disk   *cmdDisk
 
-	flagWipe    bool
-	flagEncrypt bool
-	walDevice   string
-	walEncrypt  bool
-	walWipe     bool
-	dbDevice    string
-	dbEncrypt   bool
-	dbWipe      bool
+	flagWipe       bool
+	flagEncrypt    bool
+	walDevice      string
+	walEncrypt     bool
+	walWipe        bool
+	dbDevice       string
+	dbEncrypt      bool
+	dbWipe         bool
+	flagAllDevices bool
 }
 
 func (c *cmdDiskAdd) Command() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "add <SPEC>",
 		Short: "Add a new Ceph disk (OSD)",
 		Long: `Add a new Ceph disk (OSD) for <SPEC>.
-    SPEC is either a path to a block device such as /dev/sdb or a specification for a loop file.
+    SPEC is either space separated paths to block devices or a single specification for loop files.
+    Multiple disk path arguments can be provided as <PATH> <PATH> <PATH> ...
```

Comment:
Hm there is some redundancy in the messaging here between "space separated paths" and "Multiple disk paths" that might confuse. We should also mention that WAL/DB devices don't work with multiple OSDs -- in general the help text could use a bit of an overhaul to be more explicit. This should also be synced up with the command ref doc

Proposing this text for the `Long` help:

----
Adds one or more new Ceph disks (OSDs) to the cluster, alongside optional devices for write-ahead logging and database management.

The command takes a <spec> which is either one or more paths to block devices such as /dev/sdb, or a specification for loop files.

For block devices, add a space separated list of paths, e.g. "/dev/sdb /dev/sdc ...". You may also add WAL and DB devices, but doing this is mutually exclusive with adding more than one OSD block device at a time.

The specification for loop files is of the form `loop,<size>,<nr>`

- `size` is an integer with `M`, `G`, or `T` suffixes for megabytes, gigabytes, or terabytes.
- `nr` is the number of file-backed loop OSDs to create.

For instance, a spec of `loop,8G,3` will create 3 file-backed loop OSDs of 8GB each.

Note that loop files can't be used with encryption nor WAL/DB devices.

Reply:
TBH, I am not even sure what I would do without you. @sabaini++ :D